### PR TITLE
Make initializer for ApolloSchemaDownloadConfiguration.HTTPHeader public

### PR DESCRIPTION
**Pull request by nsivertsen**
Fixes https://github.com/apollographql/apollo-ios/issues/1961

### CODE REVIEW

**Review by designatednerd — APPROVED** [review]
Guh, I always forget that you have to do this even for public structs. Thank you - if you can sign the CLA we can get this merged.

**Comment by AnthonyMDev** [comment]
Thanks so much for this fix @nsivertsen! Excited to merge this in once the CLA is signed!

File: Sources/ApolloCodegenLib/ApolloSchemaDownloadConfiguration.swift [hunk 1]
```diff
@@ -54,16 +54,21 @@ public struct ApolloSchemaDownloadConfiguration {
...
+
+    public init(key: String, value: String) {
+      self.key = key
+      self.value = value
+    }
   }
 
   /// How to download your schema. Supports the Apollo Registry and GraphQL Introspection methods.
-  let downloadMethod: DownloadMethod
+  public let downloadMethod: DownloadMethod
```
Comment (calvincestari) [inline Sources/ApolloCodegenLib/ApolloSchemaDownloadConfiguration.swift:65]: There's no reason to keep these private; once you've created the struct you should be able to inspect the property values.

File: Tests/ApolloCodegenTests/ApolloSchemaPublicTests.swift [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import XCTest
 import ApolloTestSupport
 import ApolloCodegenTestSupport
-@testable import ApolloCodegenLib
+import ApolloCodegenLib
```
Comment (calvincestari) [inline Tests/ApolloCodegenTests/ApolloSchemaPublicTests.swift:4]: Changed how this test imports `ApolloCodegenLib` to ensure `public` access is caught in further tests.

**Review by calvincestari — APPROVED** [review]

**Review by AnthonyMDev — APPROVED** [review]

File: Sources/ApolloCodegenLib/ApolloSchemaDownloadConfiguration.swift (hunk 1 above)
Comment (designatednerd) [reply Sources/ApolloCodegenLib/ApolloSchemaDownloadConfiguration.swift:65]: True for as long as they remain `let`s - If they're `var`s we may or may not want to allow direct access, although we could handle that with a `private(set)`.

**Review by designatednerd — APPROVED** [review]
SLGTM

**Pull request merged**